### ISSUE TITLE
get_task_settings('Cross Section') does not include variable 

### ISSUE THREAD

**Issue by pmendes** [copasi/basico#41]
The Cross section method depends on a variable that is tracked. The get_task_settings for 'Cross Section' does not return the variable selected, thus the task definition is incomplete

**Comment by fbergmann** [comment]
Indeed, there is currently no direct support for the cross section task in basico. And the settings returned, just contain the values from the parameter group, which are not sufficient to parameterize it.

**Comment by fbergmann** [comment]
Looking at this, I agree, since only the CNs are missing these should be added first.

**Comment by pmendes** [comment]
Thanks for this. However when there is no SingleVariable chosen this is what is returned for 'problem':
`{'AutomaticStepSize': False, 'StepNumber': 100, 'StepSize': 0.01, 'Duration': 1.0, 'TimeSeriesRequested': True, 'OutputStartTime': 0.0, 'Output Event': False, 'Start in Steady State': False, 'Use Values': False, 'Values': '', 'LimitCrossings': False, 'NumCrossingsLimit': 0, 'LimitOutTime': False, 'LimitOutCrossings': False, 'PositiveDirection': True, 'NumOutCrossingsLimit': 0, 'LimitUntilConvergence': False, 'ConvergenceTolerance': 1e-06, 'Threshold': 0.0, 'DelayOutputUntilConvergence': False, 'OutputConvergenceTolerance': 1e-06, 'SingleVariable': }
`
note the end: `'SingleVariable': }`

there is nothing assigned to this dictionary entry, but it also does not return None, so I am unsure how to test for it being empty (my ignorance of python is showing here). For example:

`if( cs['problem']['SingleVariable'] is not None):
    set_task_settings( etc )
`
it still executes the if with that result

**Comment by fbergmann** [comment]
I've tried this here with a file that has cross section setup and get: 

```yaml
{'AutomaticStepSize': False, 'StepNumber': 100, 'StepSize': 0.01, 'Duration': 1000.0, 'TimeSeriesRequested': True, 'OutputStartTime': 0.0, 'Output Event': False, 'Start in Steady State': False, 'Use Values': False, 'Values': '', 'LimitCrossings': True, 'NumCrossingsLimit': 50, 'LimitOutTime': False, 'LimitOutCrossings': True, 'PositiveDirection': False, 'NumOutCrossingsLimit': 40, 'LimitUntilConvergence': True, 'ConvergenceTolerance': 1e-05, 'Threshold': 0.0, 'DelayOutputUntilConvergence': True, 'OutputConvergenceTolerance': 1e-05, 'SingleVariable': 'X2.Rate'}
```

could you send me the file you are having issues with?

**Comment by pmendes** [comment]
Ah, the problem may be that set_task_settings does not seem to understand 'SingleVariable' in the problem... 
If I have a variable defined, then 'SingleVariable' returns the proper value (like 'Values[x]') but trying to set it with set_task_settings results in 
`could not set value Values[x] for parameter SingleVariable`

**Comment by pmendes** [comment]
[see relevant code here: ](https://github.com/pmendes/copasi-utilities/blob/916ecb74c0f477413d0fdddf9e7ee51eda92a308/MEC.py#L504C1-L509C13)

**Comment by fbergmann** [comment]
I've released 0.63, that takes care of it. Unset CNs will now be displayed as `''`, and settings ought to work as well.

**Comment by pmendes** [comment]
Works for me, this is resolved.
